governance.md

```diff
@@ -0,0 +1,164 @@
+
+To stand up this Governance for a 1.0 good-enough minimal-viable-governance (MVG), we are looking to our key predecessors for some of our starting points.
+As an initial set of principles we are combining elements of the [Kubernetes community principles](https://github.com/kubernetes/community/blob/master/governance.md#principles) and the [OpenInfra Foundations Four Opens](https://openinfra.dev/four-opens/).
+
+#### Op1st Community Principles 1.0
+Our mission requires us to use open source principles in ways beyond licensing the code we write, but at the core we are an open source community and want to adhere to the best possible principles for creating an open, welcoming, and inclusive community.
+
+These are the Op1st community principles we adhere to:
+1. **Open Source** : Any content we create is released under an open source license. Content includes software, documentations, configuration files, and training materials.
+  - This is crucial to fulfilling our mission around open sourcing operations
+2. **Welcoming and respectful** : We intend to be an open, welcoming, and inclusive community.
+Diversity is our strength.
+We have and follow a Code of Conduct.
+  - We currently follow the OpenInfra Foundation Code of Conduct as a member of the OpenInfra Labs.
+3. **Open Design** : We follow a transparent and open process for planning and designing any content, services, and features of Op1st.
+Ideas and contributions are accepted according to their technical merit and alignment with project objectives, scope, and design principles.
+4. **Open Work** : Two of our core tools for successful community building are transparency and accessibility.
+To support that, work and collaboration must be done in public to the greatest degree possible.
+Refer to the section on Community Groups and SIGs in particular for details on how this is performed.
+5. **Truly Open Community** : The best way to build a sustainable open source community is to ensure the community is a cohesive, inclusive, and equitable place where all the voices are heard and anyone can rise to leadership positions.
+
+### Code of Conduct
+
+We are using the [upstream OpenInfra Foundation Code of Conduct](https://openinfra.dev/legal/code-of-conduct) until we can convene the new Community SIG to lead an open effort to create our own, modern code of conduct.
+
+### Community membership
+**Note:** This document is a work in progress, especially these membership guidelines
+
+We are opening the Governance 1.0 directly borrowing from the [Kubernetes community group definitions](https://github.com/kubernetes/community/blob/master/community-membership.md), then reviewing and refining for the Governance 1.1.
+
+This doc outlines the various responsibilities of contributor roles in Op1st.
+The Op1st project is subdivided into subprojects under SIGs.
+Responsibilities for most roles are scoped to these subprojects.
```

Comment:
Maybe link https://stumblingabout.com/2011/01/27/ownership-roles-and-responsibilities/ 
This is also linked from the k8s governance files. It explains the interaction of those roles quite well

Reply:
I totally agree with the concept, but I always hesitate to link to external, potentially ephemeral URLs. That is, if we rely upon that link for the understanding without having our own local, refined version, we are rudderless without that person's domain being up and running. (I know, I know, let's not talk about github.com, quay.io, etc. etc.)

As an interim good-enough, I'll make a change that identifies it as a separate reference and we can put it on the list to update for 1.1. (One way may to look for the gaps in the [Open Source Way content](https://www.theopensourceway.org/the_open_source_way-guidebook-2.0.html#_community_roles) and that link above, then fix the OSW guidebook with content to fill the gaps.)

Reply:
```suggestion
Responsibilities for most roles are scoped to these subprojects.
"[Ownership, roles and responsibilities](https://stumblingabout.com/2011/01/27/ownership-roles-and-responsibilities/)" is a good external reference for understanding how the project roles interact.
```